src/js/languages/default.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Broad question, how would this file work when the grommet user isn't using English?

Reply:
It would still be the default. This isn't dependent on browser locale. Eric and I talked a bit about whether grommet itself would have localizations for it's messages and it's our expectation that whomever had an app that was being localized would at that point provide localized versions of the messages. BTW, one rule of I18N/L10N is to avoid having any mixed languages. If we provided any localizations that got triggered by browser locale or such, that could result in grommet messages being in one language while the app's content was maybe in another language. Best to make sure the app controls all that.